accounting/bank/feeds/ponto.rst

```diff
@@ -0,0 +1,115 @@
+======================================
+Ponto as Bank Synchronization provider
+======================================
+
+**Ponto** is a service that allows companies and professionals to aggregate their accounts in one
+place and directly see all their transactions within one app.
+
+It is a third-party solution that is continuously expanding the number of bank institutions that can
+be synchronized with Odoo.
+
+.. image:: media/ponto_logo.png
+   :align: center
+   :alt: Logo of the Ponto brand
+
+**Odoo Accounting** can synchronize directly with your bank to get all bank statements imported
+automatically into your database. This allows for easier **bank reconciliation**.
+
+When :doc:`adding a bank account on Odoo <../setup/create_bank_account>`, you can see if your bank
+requires a connection through Ponto by searching for your bank institution, and clicking on it.
```

Comment:
Break the line but not the paragraph: https://www.odoo.com/documentation/user/13.0/contributing/documentation/rst_cheat_sheet.html#break-the-line-but-not-the-paragraph